bundles/InstallBundle/InstallerKernel.php

```diff
@@ -52,14 +52,6 @@ public function getProjectDir()
         return $this->projectRoot;
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getRootDir()
```

Comment:
please revert this also and mark it deprecated.

Reply:
`InstallerKernel` is `@internal`

Reply:
Nevermind. Just noticed it it marked as `internal`